content/tutorials/first-aws-environment.md

```diff
@@ -0,0 +1,210 @@
+  - If you haven't set this up before, check out [the Geodesic how-to on authenticating with aws-vault]({{< relref "howto/geodesic/authenticate-with-aws-vault.md" >}}).
+  - We'll refer to your `aws-vault` profile through out this tutorial as `$YOUR_PROFILE`. When you see that, be sure to replace with whatever name you added to `aws-vault`.
+
+### Understanding
+
+Prior to starting this tutorial, you should be sure you understand [our various concepts and terminology]({{< relref "fundamentals/concepts.md" >}}) and have gone through our [Getting started with Geodesic]({{< relref "tutorials/geodesic-getting-started.md" >}}) and [Getting started with Atmos tutorial]({{< relref "tutorials/atmos-getting-started.md" >}}) because we'll be building upon the knowledge in both of those tutorials.
+
+## Tutorial
+
+### 1. Clone the tutorials repository + Run the `tutorials` image
+
+As part of this tutorial (and others in our tutorial series), we'll be utilizing [our tutorials repository](https://github.com/cloudposse/tutorials). This repository includes code and relevant materials for you to use alongside this tutorial walk through.
+
+Let's clone it to your local machine and `cd` into it:
+
+```bash
+git clone git@github.com:cloudposse/tutorials.git
+
+cd tutorials
+```
+
+Now that we've got our code, we're going to want to interact with it using our standard set of tools. Following the SweetOps methodology, we're going to so via a Docker image built on top of Geodesic. This entire `tutorials` repository is actually Dockerized to make that part easy, so let's build and run our `cloudposse/tutorials` image:
+
+```bash
+# Geodesic is the base image for our tutorials image and it isn't tiny by any means
+# Go grab a coffee while you wait ☕️
+make build
+
+make run
+```
+
+This will build the `tutorials` image locally on your machine and then runs it while also mounting the various tutorial folders so you can edit them on your host machine or in the container and the changes will propogate either direction.
+
+![Tutorial Shell](/assets/tutorials-3-tutorials-shell.png)
+
+Now that we're running in our container, let's get into our specific tutorial:
+
+```bash
+cd /tutorials/03-first-aws-environment
+```
+
+This `03-first-aws-environment/` directory that you're now in should be looking a little bit familiar from our first tutorial:
+
+* We've got a `components/` directory which holds terraform components that we're looking to `apply` to our AWS Account.
+* We've got a `stacks/` directory which holds the declarative descriptions of 3 environments:
+  * `uw2-dev.yaml`: Our dev environment hosted in us-west-2.
+  * `uw2-prod.yaml`: Our prod environment hosted in the same region.
+  * `ue2-root.yaml`: Our global resources hosted in us-east-2.
+* We've also got a `bin/unquify.sh` file that we'll discuss shortly.
```

Comment:
WHat is `unquify` in reference to? I don't it =)

Reply:
Haha will fix -- Thanks for the catch.

Reply:
Fixed.